src/CarbonAware.WebApi/src/Models/EmissionsForecastDTO.cs

```diff
@@ -26,7 +26,7 @@ public record EmissionsForecastDTO : EmissionsForecastBaseDTO
     /// }
     /// </example>
     [JsonPropertyName("optimalDataPoint")]
```

Comment:
_If changing the contract:_

```suggestion
    [JsonPropertyName("optimalDataPoints")]
```